src/envs/stick_button.py

```diff
@@ -576,3 +578,29 @@ def _AboveNoButton_holds(self, state: State,
             if self._Above_holds(state, [stick, button]):
                 return False
         return True
+
+    def get_event_to_action_fn(
+            self) -> Callable[[State, matplotlib.backend_bases.Event], Action]:
+        logging.info("Controls: mouse click to move, any key to press")
+
+        def _event_to_action(state: State,
+                             event: matplotlib.backend_bases.Event) -> Action:
+            if event.key is not None:
+                # Press action.
+                return Action(np.array([0.0, 0.0, 0.0, 1.0], dtype=np.float32))
+            # Move action.
+            rx = state.get(self._robot, "x")
+            ry = state.get(self._robot, "y")
+            tx = event.xdata
+            ty = event.ydata
+            assert tx is not None and ty is not None, "Out-of-bounds click"
+            # Move toward the target.
+            dx = np.clip(tx - rx, -self.max_speed, self.max_speed)
+            dy = np.clip(ty - ry, -self.max_speed, self.max_speed)
+            # Normalize.
+            dx = dx / self.max_speed
+            dy = dy / self.max_speed
+            # No need to rotate or press.
```

Comment:
somewhere around here you should `assert CFG.stick_button_disable_angles`

Reply:
oops thanks, had this before but it got lost somehow